internal/credential/store.go

```diff
@@ -19,6 +19,8 @@ import (
 	credentials "github.com/oras-project/oras-credentials-go"
 )
 
+var CreateNewStore = credentials.NewStore
```

Comment:
Any reason to export this variable?

Reply:
No, my mistake. I will revert these changes.